bash/scripts/find_required_commits.sh

```diff
@@ -4,8 +4,8 @@ set -eo pipefail
 # find-required-commits parses a commit description for any required sibling
 # commits to pass along to downstream job(s)...
 find-required-commits() {
-  local git_commit="${1}"
-  local commit_description="${COMMIT_DESCRIPTION:-$(git log --format=%B -n 1 "${git_commit}")}"
+  git_commit="${1}"
```

Comment:
@sgoings http://tldp.org/LDP/abs/html/localvar.html#EXITVALANOMALY01 clued me in to weird behavior that can result -- though that example doesn't seem to be related to our use here, I noticed in the prev version `local var=value` that the finding of required commits wouldn't run correctly (at some point the values weren't available?  I never really found a concrete answer).  When `local` was removed, the logic worked again as expected.